### PR TITLE
[GJ-75] Unify function names

### PR DESCRIPTION
**Pull request by rui-mo**
This pr unified the function names according to the Substrait specifications in:
https://substrait.io/extensions/.

Based on: https://github.com/substrait-io/substrait/pull/147.
Closes https://github.com/oap-project/gazelle-jni/issues/75.

### CODE REVIEW

**Comment by rui-mo** [comment]
@marin-ma Hi Rong, please help to check if this PR works for you, thanks.

File: jvm/src/main/scala/com/intel/oap/expression/AggregateFunctionsBuilder.scala [hunk 1]
```diff
@@ -17,19 +17,20 @@
...
   def create(args: java.lang.Object, aggregateFunc: AggregateFunction): Long = {
     val functionMap = args.asInstanceOf[java.util.HashMap[String, java.lang.Long]]
     aggregateFunc match {
-      case sum: Sum => {
-        ExpressionBuilder.newScalarFunction(functionMap, "SUM")
-      }
-      case other => null
+      case sum: Sum =>
+        val functionName = ConverterUtils.makeFuncName(
+          "sum", Seq(sum.child.dataType), FunctionConfig.OPT)
```
Comment (zzcclp) [inline jvm/src/main/scala/com/intel/oap/expression/AggregateFunctionsBuilder.scala:31]: suggestion: All the function names are defined as a final static constants.

Comment (rui-mo) [reply jvm/src/main/scala/com/intel/oap/expression/AggregateFunctionsBuilder.scala]: fixed in the latest commit.

**Comment by zzcclp** [comment]
Verified with ClickHouse Lib, LGTM.

**Pull request merged**